packages/data-store/src/entities/issuanceBranding/ImageDimensionsEntity.ts

```diff
@@ -0,0 +1,22 @@
+import { BaseEntity, Column, Entity, PrimaryGeneratedColumn } from 'typeorm'
+import { IBasicImageDimensions } from '../../types'
+
+@Entity('ImageDimensions')
+export class ImageDimensionsEntity extends BaseEntity {
+  @PrimaryGeneratedColumn('uuid')
+  id!: string
+
+  @Column({ name: 'width', nullable: false, unique: false })
+  width!: number
```

Comment:
See other remark. Probably it makes sense to change these to string, if we want to support other dimension types in the future (proportional, percentage etc)

Reply:
Okay, guess this always comes from the absolute dimensions of an image, so scratch that